scripts/tools/nrfconnect/generate_nrfconnect_chip_factory_data.py

```diff
@@ -61,6 +64,82 @@ def get_raw_private_key_der(der_file: str, password: str):
         return None
 
 
+def gen_test_certs(chip_cert_exe: str, output: str, vendor_id: int, product_id: int, device_name: str):
+    CD_PATH = MATTER_ROOT + "/credentials/test/certification-declaration/Chip-Test-CD-Signing-Cert.pem"
+    CD_KEY_PATH = MATTER_ROOT + "/credentials/test/certification-declaration/Chip-Test-CD-Signing-Key.pem"
+    PAA_PATH = MATTER_ROOT + "/credentials/test/attestation/Chip-Test-PAA-NoVID-Cert.pem"
+    PAA_KEY_PATH = MATTER_ROOT + "/credentials/test/attestation/Chip-Test-PAA-NoVID-Key.pem"
+
+    attestation_certs = namedtuple("attestation_certs", ["dac_cert", "dac_key", "pai_cert"])
+
+    log.info("Generating new certificates using chip-cert...")
+
+    # generate Certifiation declaration
+    cmd = [chip_cert_exe, "gen-cd",
+           "--key", CD_KEY_PATH,
+           "--cert", CD_PATH,
+           "--out", output + "/CD.der",
+           "--format-version",  str(1),
+           "--vendor-id",  hex(vendor_id),
+           "--product-id",  hex(product_id),
+           "--device-type-id", "0xA",
+           "--certificate-id", "ZIG20142ZB330003-24",
+           "--security-level",  str(0),
+           "--security-info",  str(0),
+           "--certification-type",  str(0),
+           "--version-number", "0x2694",
+           ]
+    subprocess.run(cmd)
```

Comment:
nit: we could use the `CompletedProcess` returned by the `subprocess` to check the result of the call.